packages/core/gulpfile.js

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+const gulp = require('gulp');
+const serverDir = './src/common/i18n/';
+const extensionDir = 'lib/common/i18n';
+
+gulp.task('copy_json_files', () => {
+    gulp.src(serverDir + '*.json')
+        .pipe(gulp.dest(extensionDir))
+});
+
+
+
+
+
+
```

Comment:
Remove all gulpfile.js, replace it with the following command
"shx mkdir -p lib/common/i18n  && shx cp ./src/common/i18n/*.json lib/common/i18n/"

for each package where we use Globalize